jicamargo/space-travelers
# Jest Testing functions (Rockets)

# Jest Testing functions (Rockets)

This PR adds test files for:

- Rockets and Missions slices in the Redux store.
- Navigation
- Rockets
- RocketCard
- RocketProfile

Testing is an essential part of ensuring the stability and reliability of the application. By adding these tests, we can verify that the Rockets and Missions slices in the Redux store are functioning correctly, making it easier to catch and prevent potential issues.

## How to Test

1. Ensure that all dependencies are installed by running `npm install` or `yarn install`.
2. Run the tests with the command `npm test` or `yarn test`.
3. Verify that all the tests in the `RocketsSlice.test.js` and `MissionsSlice.test.js` files pass successfully.
4. Manually test the Rockets and Missions functionality in the application to ensure that the tests cover all relevant scenarios.

Please review and merge this PR. Thank you!
